src/ResourceManager/ServiceBus/Commands.ServiceBus/help/Get-AzureRmServiceBusDRConfigurations.md

```diff
@@ -0,0 +1,119 @@
+---
+external help file: Microsoft.Azure.Commands.ServiceBus.dll-Help.xml
+Module Name: AzureRM.ServiceBus
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmServiceBusDRConfigurations
+
+## SYNOPSIS
+Retrieves Alias(Disaster Recovery configuration) for primary or secondary namespace
+
+## SYNTAX
+
+```
+Get-AzureRmServiceBusDRConfigurations [-ResourceGroupName] <String> [-Namespace] <String>
+ [[-Name] <String>] [-DefaultProfile <IAzureContextContainer>]
+```
+
+## DESCRIPTION
+The **Get-AzureRmServiceBusDRConfigurations** Retrieves Alias(Disaster Recovery configuration) for primary or secondary namespace
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzureRmServiceBusDRConfigurations -ResourceGroupName "SampleResourceGroup" -Namespace "SampleNamespace_Primary" -Name "SampleDRCongifName"
+```
+
+Retrieves alias "SampleDRCongifName" configuration for primary namespace "SampleNamespace_Primary"
+
+### Example 2
+```
+PS C:\> Get-AzureRmServiceBusDRConfigurations -ResourceGroupName "SampleResourceGroup" -Namespace "SampleNamespace_Primary"
+```
+
+Retrieves all alias configurations(list) for primary namespace "SampleNamespace_Primary"
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Name
+DR Configuration Name.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: 2
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Namespace
+Namespace Name.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+Resource Group Name.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+## INPUTS
+
+### System.String
+
+
+## OUTPUTS
+
+Name              : SampleDRCongifName
```

Comment:
@v-Ajnava please move this to the examples section above 😃

Reply:
Resovled